NAMESPACE

```diff
@@ -51,6 +51,7 @@ S3method(cube, data.table)
 S3method(rollup, data.table)
 export(frollmean)
 export(frollsum)
+export(frollmax)
```

Comment:
@jangorecki should we roll back #5889 before merging?

Reply:
Probably better

Reply:
OK, I hope it doesn't do anything too catastrophic with all the `--force` pushes going around. Since the diffs are still visible in the #5889 web UI, at worst we can reconstitute the changes manually.

Reply:
NB: had to remove the #5910 changes to src/froll.c. Will need to reconstitute those.